src/include/parser/statement.h

```diff
@@ -38,6 +41,7 @@ class Statement {
 
 private:
     common::StatementType statementType;
+    bool skipResult_;
```

Comment:
I would call this `internal_`. And also comment, internal statements are those rewritten from input and whose result should not be rendered on client side.